Misc/NEWS.d/next/Windows/2019-08-06-18-09-18.bpo-25172.Akreij.rst

```diff
@@ -0,0 +1 @@
+The crypt module is not supported on Windows. If anybody tries to use it by importing the module, they will see an ImportError with a message saying its not supported on Windows. One will also see an appropriate error message on any other platform if the crypt module is not found or built.
```

Comment:
I'd word this as:

> Trying to import the crypt module on Windows will result in an ImportError with a message explaining that the module isn't supported on Windows. On other platforms, if the underlying _crypt module is not available, the ImportError will include a message explaining the problem.

Reply:
I have updated the blurb based on your recommendation. Thank you.